server/command.go

```diff
@@ -221,6 +221,22 @@ func (p *Plugin) executeLinkCommand(args *model.CommandArgs, parameters []string
 		return p.cmdError(args.UserId, args.ChannelId, "Error occurred while saving the subscription")
 	}
 
+	if _, err = p.API.ShareChannel(&model.SharedChannel{
+		ChannelId: channelLink.MattermostChannelID,
+		TeamId:    channelLink.MattermostTeamID,
+		Home:      true,
+		ReadOnly:  false,
+		CreatorId: p.userID,
+		RemoteId:  p.remoteID,
+		ShareName: channelLink.MattermostChannelID,
+	}); err != nil {
+		if err2 := p.store.DeleteSubscription(channelsSubscription.ID); err2 != nil {
+			p.API.LogDebug("Unable to rollback the subscription creation", "channelID", channelLink.MattermostChannelID, "error", err2.Error())
+		}
+		p.API.LogDebug("Unable to share the channel", "channelID", channelLink.MattermostChannelID, "error", err.Error())
```

Comment:
Let's log this as a warning:
```suggestion
		p.API.LogWarn("Unable to share the channel", "channelID", channelLink.MattermostChannelID, "error", err.Error())
```